### DISCUSSION TITLE
Allow use of URL objects with fetch

### DISCUSSION THREAD

**Discussion by marr** [remix-run/remix#2649] [Proposals]
I was reading up on the MDN fetch docs, and they say that URL objects can be used as the first parameter to fetch calls. When I try this in my local TypeScript project, I get an error:

> Argument of type 'URL' is not assignable to parameter of type 'RequestInfo'

The documentation for fetch I'm referring to is here: https://devdocs.io/dom/fetch#parameters.
Maybe this is more a `node-fetch` feature request, let me know.

Thanks!

**Comment by sergiodxa** [comment]
I think this is an issue with node-fetch, Remix uses that and the URL object comes from Node/the browser.
